tools/nmr_bucketing/NmrBucketing_xml.xml

```diff
@@ -0,0 +1,302 @@
++---------------------------+------------+
+| 1 : Choose your inputs    |   tsv      |
++---------------------------+------------+
+
+**Choose your inputs**
+
+You have three methods for your inputs:
+
+| Zip file (recommended): You can put a zip file containing your inputs as raw Bruker files: myinputs.zip (containing all your conditions as sub-directories).
+| Tsv file: You can put a tsv file containing your inputs as preprocessed spectra: myinputs.tsv (containing all your conditions in columns and chemical shifts in rows).
+
+.. image:: ./static/images/Mth_Architecture_Repertoire_Bruker.png
+:width: 800
+
+----------
+Parameters
+----------
+
+Bucket width
+| size of windows
+|
+
+Left limit
+| Upper boundary: values greater than this value are not used in the bucketing. Default value is 10.0 ppm
+|
+
+Right limit
+| Lower boundary: values lower than this value are not used in the bucketing. Default value is 0.5 ppm
+|
+
+Exclusion zone(s)
+| Spectral regions to exclude, water, solvents, ... resonance
+| If YES: parameters **Lower exclusion zone** and **Upper exclusion zone** are visible,
+| If NO: no zone to exclude
+| Default value is NO
+|
+
+Left exclusion zone
+| Upper boundary of exclusion zone
+|
+
+Right exclusion zone
+| Lower boundary of exclusion zone
+
+| *Notes:*
+| - these parameters can be used several times using the "Add new exclusion zones" button
+|
+
+Spectra representation:
+| Graphical chart of bucketed and integrated raw files
+| If "Overlay": the n (sample number) spectra are overlaid on the same figure
+| If "One_per_individual": pdf file includes n pages (1 per sample)
+|
+
+
+------------
+Output files
+------------
+
+
+bucketedData.tsv
+| tabular output
+| Data matrix with p rows (buckets) and n columns (samples) containing the intensities
+|
+
+sampleMetadata.tsv
+| tabular output
+| file with n rows (samples) and 2 columns containing sample identifier (rownames) and sample order: the rownames of sampleMetadata must be identical to the colnames of the bucketedData. Can add columns with numeric and/or character sample metadata. This file is optional in the normalization step and mandatory in the statistical analysis step of the workflow.
+|
+
+variableMetadata.tsv
+| tabular output
+| file with p rows (buckets) and 2 columns containing variable identifier (rownames) and bucket order: the rownames of variableMetadata must be identical to the rownames of the bucketedData. Can add columns with numeric and/or character variable metadata. This file is mandatory in the statistical analysis step of the workflow.
+|
+
+spectra.pdf
+| pdf output
+| Graphical chart of bucketed and integrated data
+|
+
+
+---------------------------------------------------
+
+---------------
+Working example
+---------------
+
+
+.. class:: warningmark
+
+Under construction
+
+.. image:: ./static/images/Mth_Travaux.png
+:width: 100
+
+---------------------------------------------------
+
+Changelog/News

```

Comment:
I would remove the changelog its already in the Readme file and its hard to keep it in sync.